packages/vscode-extension/package.nls.json

```diff
@@ -285,6 +285,10 @@
     "teamstoolkit.localDebug.portWarning": "Changing port(s) in package.json may break local debug. Ensure all port changes are expected or refer to documentation by clicking `Learn More` button. (%s package.json location: %s)",
     "teamstoolkit.localDebug.prerequisitesCheckFailure": "Prerequisites Check Failed. Please check the %s to see the details. If you wish to bypass checking and installing any prerequisites, you can disable them in Visual Studio Code settings.",
     "teamstoolkit.localDebug.npmInstallFailure": "NPM Package Install Task Failed. Please check the %s to see the details.",
+    "teamstoolkit.localDebug.setUpTabFailure": "Set up Tab Task Failed. Please check the %s to see the details.",
+    "teamstoolkit.localDebug.setUpBotFailure": "Set up Bot Task Failed. Please check the %s to see the details.",
+    "teamstoolkit.localDebug.setUpSSOFailure": "Set up SSO Task Failed. Please check the %s to see the details.",
+    "teamstoolkit.localDebug.prepareManifestFailure": "Prepare manifest Task Failed. Please check the %s to see the details.",
```

Comment:
Are these strings used? Seems many of the words are capitalized, is it expected?

Reply:
`setUpTabFailure` is used in this PR and the others will be used in following PRs.